openc3-cosmos-init/plugins/packages/openc3-cosmos-demo/targets/INST/lib/cmd_validator.rb

```diff
@@ -0,0 +1,35 @@
+# encoding: ascii-8bit
+
+# Copyright 2024 OpenC3, Inc.
+# All Rights Reserved.
+#
+# This program is free software; you can modify and/or redistribute it
+# under the terms of the GNU Affero General Public License
+# as published by the Free Software Foundation; version 3 with
+# attribution addendums as found in the LICENSE.txt
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+
+# This file may also be used under the terms of a commercial license
+# if purchased from OpenC3, Inc.
+
+require 'openc3/packets/command_validator'
+
+class CmdValidator < OpenC3::CommandValidator
```

Comment:
Name is kinda generic.  Should probably Include the target name in the class name.  InstCmdValidator.